.github/workflows/int_test.yml

```diff
@@ -71,8 +76,10 @@ jobs:
         tenant-id: ${{ secrets.AZURE_TENANT_ID }}
         subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
     - name: "Create Key Vault for clusters"
+      id: create-keyvault
```

Comment:
nit: probably don't need an id on this step if we're not referencing it anywhere later